README.md

```diff
@@ -43,9 +43,11 @@ You will need to create the following secret. Use the credentials generated in t
 |--------------|------------------------------------------------------|
 |zuliprc       | The file content of the zuliprc obtained from step 2 |
 
-### Step 4 - Enable GitHub Pages
+### Step 4 - Enable GitHub Pages or set up base URL
 
-Go to `https://github.com/<username>/<repo-name>/settings/pages`, select `main` (or a branch of your choosing), and `/` as the folder. Save the changes.
+Go to `https://github.com/<username>/<repo-name>/settings/pages`, select `main` (or a branch of your choosing), and `/` as the folder. Save the changes. The base URL of the generated site will be resolved to GitHub Pages, i.e., `https://<username>.github.io/<repo-name>` or the configured custom domain name.
+
+Alternatively, you can configure the `base_url` option to populate the base URL without enabling GitHub Pages.
```

Comment:
The "without enabling GitHub Pages" is unclear to me. It could be "This option could be useful in situation when you are not using GitHub Pages."

Reply:
Updated. Please take a look.